content/future.html

```diff
@@ -2,104 +2,133 @@ <h1 id="future"> Future </h1>
 <h3> Post-graduation plans </h3>
 <p class="half">
   <span class="big">
-    As of 5 months before graduation, 77% of students already have an offer, and 15% of students are
-    still looking for a job.
+    90% of respondants had already accepted a job offer two months prior
+    to graduating.
   </span>
-  7 students are planning to do graduate studies after
-  graduating. Although most of the class plans to join the workforce immediately,
-  34% of students are considering grad school in the future.
-  <br/><br/>
-
-  Even though 80% of the class has worked in California, less than half of
-  students have chosen to live in California after graduating. Although California
-  is perceived by many as the place to be for software engineers, seeing the gap
-  in the conversion rate suggests that there are many other great places for
-  software engineers such as East Coast Canada, East Coast US and Pacific North
-  West US.
+  However, only 52 of the 85 SE graduates, and a few who transferred, answered this
+  question. The only take-away is that most students had the opportunity to
+  accept a job offer prior to graduating.
+  <!-- 85 - 52  = 33 -->
 </p>
 <div class="half second">
   <span class="graph-title">Post graduation plans</span>
   <div id="post-grad"></div>
   <div class="vb"></div>
-  <span class="graph-title">Desired location</span>
+  <span class="graph-title">Relocation plans</span>
   <div id="post-location"></div>
 </div>
 
-<div class="vbx2"></div>
+<div class="vbx3"></div>
 
-<h3> Motivations </h3>
 <p class="half">
   <span class="big">
-    Good compensation and growth as an engineer were the main
-    factors affecting post-graduation plans.
+    The co-op acts as a great pipeline for hiring young talent for companies.
```

Comment:
```suggestion
    The co-op experience acts as a great pipeline for hiring young talent for companies.
```

or just "Co-op acts as [...]"